reference/commands/creator/export-pkg.rst

```diff
@@ -176,8 +176,14 @@ with the ``export-pkg`` command. Check this :ref:`How to package existing binari
       $ conan source . --source-folder src
       $ conan install . --install-folder build_x86 -s arch=x86
       $ conan build . --build-folder build_x86 --source-folder src
-      $ conan export-pkg . Hello/0.1@user/stable --build-folder build_x86
+      $ conan export-pkg . Hello/0.1@user/stable --build-folder build_x86 -s arch=x86
```

Comment:
Should we recommend this as default or only for special cases as warned below?

Reply:
This is the default one, it is everywhere in the docs. There is no export-pkg without using settings or profile anymore.